LEGO1/mxbackgroundcolor.cpp

```diff
@@ -0,0 +1,22 @@
+#include "mxbackgroundcolor.h"
+#include "mxstring.h"
+
+// OFFSET: LEGO1 0x1003bec0
+MxBackgroundColor::~MxBackgroundColor()
+{
+
+  delete &m_colorString;
+  delete &m_name;
+}
+
+// OFFSET: LEGO1 0x1003bea0
+MxString *MxBackgroundColor::GetColorString()
+{
+  return &m_colorString;
+}
+
+// OFFSET: LEGO1 0x1003beb0
+void MxBackgroundColor::SetColorString(const char *colorString)
```

Comment:
I doubt the names of this class.
These behave more like variables with callbacks.

The `GetName` method is used in 3 classes:
```
  100d7498, 100d74a8, 100d74b8, 100d8698
  100d86a8, 100d86b8, 100d86c8, 100da878
```
These vtables have the 1st and 3rd member in common, only the 2nd differs (which accepts a `const char*` and modifies the global state)